apps/web/src/global.css

```
@@ -0,0 +1,59 @@
+#app {
+  min-height: 100%;
+  margin: 0;
+}
+
+body {
+  background: var(--fasti-background);
+  color: var(--fasti-foreground);
+  font-family: var(--fasti-font-body);
+  font-size: 1rem;
+  line-height: 1.5;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)